js/npm/api-keys/createKey.js

```diff
@@ -17,7 +17,7 @@ async function createApiKeySample() {
         featureFlagOne: true,
         featureFlagTwo: false,
       },
-      expiry: new Date("2023-09-01 00:00:00"), // optional expiry date
+      expiry: "2023-09-01 00:00:00", // optional expiry date
```

Comment:
This one works, but I'm curios on why you're favoring expiry as string instead of a date object :thinking:

Reply:
The date object was causing an error when posting to the server. I removed it for speed!